docs/pages/blog/embracing-the-future.md

```diff
@@ -0,0 +1,128 @@
+
+## Exploring the Different ICP Wallet Standards
+
+### ICRC-25 Base Signer Interaction Standard
+
+This standard outlines a foundational protocol for communication between decentralized apps (dapps) and signers. It specifies a set of messages to facilitate this interaction, ensuring a seamless and secure exchange of information. The ICRC-25 aims to create a uniform language for dapps and signers, enhancing interoperability and simplifying the development process.
+
+#### Establish a Signer Connection:
+
+```typescript
+import * as IdentityKit from '@nfid/identity-kit'
+import { NFID, II } from '@nfid/identity-kit/adapter'
+
+const nfid = NFID.config({
+  providerUrl: 'https://nfid.one',
+})
+const ii = II.config({
+  providerUrl: 'https://identity.ic0.app',
+})
+
+IdentityKit.config({
+  derivationOrigin: 'https://<canisterId>.ic0.app',
+  appMeta: {
+    name: 'My App',
+    logo: 'https://domain.com/path/to/logo.png',
+  },
+})
+
+// Establish a connection with the Signer the user has selected:
+const connection = await IdentityKit.connect({
+  signer: nfid,
+})
+```
+
+#### Exploring available standards on a given signer:
+
+```typescript
+type Response = string[]
+
+const supported: SupportedStandards = await connection.supportedStandards()
+// ["ICRC-31", "ICRC-32"]
+```
+
+### ICRC-31 Get Principals
+
+The `icrc31_get_principals` message plays a crucial role in identity management. It allows applications to retrieve information about the identities managed by a signer, streamlining the authentication process. This standard is pivotal in ensuring that users can manage their digital identities efficiently and securely.
+
+```typescript
+type Response = {
+  version: string // "1"
+  principals: string[] // ["gyu2j-2ni7o-o6yjt-n7lyh-x3sxq-zh7hp-sjvqe-t7oul-4eehb-2gvtt-Jae"]
+}
+
+const response: Response = await connection.getPrincipals()
+```
+
+### ICRC-32 Sign Challenge
+
+The `icrc32_sign_challenge` method provides a secure mechanism for verifying ownership of user identities. By facilitating cryptographic proofs, this standard strengthens the security framework of applications, ensuring that user identities are protected and authenticated reliably.
+
+```typescript
+type Response = {
+  version: string // "1"
+  principals: string[] // ["gyu2j-2ni7o-o6yjt-n7lyh-x3sxq-zh7hp-sjvqe-t7oul-4eehb-2gvtt-Jae"]
+}
+
+const response: Response = await connection.getPrincipals()
+```
+
+### ICRC-33 Call Canister
+
+This standard allows applications to request the execution of calls to third-party canisters by the signer. By using the `icrc33_call_canister` method, applications can interact with a broader range of services and functionalities, enhancing the versatility and reach of ICP-based applications.
```

Comment:
I think this is underselling the benefits:
* applications get to make canister calls under a _shared_ identity, some example use-cases would definitely help (in this section or further above)
  * i.e. have all your ICP in one / only a few place(s)
  * control a canister using a singer, and have various other applications help you manage it (i.e. a canister wasm installation helper or something 🤷 ) _without_ them being a controller
  * a voting application that does _not_ need to be a hot-key controller of your neurons
  * ...
* it is an implementation of the transaction approval flow, which gives the user much more control over the actions taken with a given identity